### PR TITLE
Added some more hypixel ez

### PR DESCRIPTION
**Pull request by natan515**
An updated version of them.

**Describe the pull**
This is an updated version of hypixel EZ

**Describe how this pull is helpful**
It's nice to be updated and have more funny sentences. 

**Additional context**
Got the info from a form post.

### CODE REVIEW

File: src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoEZ.kt [hunk 1]
```diff
@@ -37,15 +37,31 @@ object AutoEZ : Module() {
         CUSTOM("");
     }
 
-    private val hypixelCensorMessages: Array<String> = arrayOf(
+     private val hypixelCensorMessages: Array<String> = arrayOf(
```
Comment (Xiaro) [inline src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoEZ.kt:40]: Fix the extra indent please

**Review by Xiaro — CHANGES_REQUESTED** [review]

File: src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoEZ.kt [hunk 2]
```diff
@@ -84,7 +84,7 @@ object AutoEZ : Module() {
             "What happens if I add chocolate milk to macaroni and cheese?",
             "Can you paint with all the colors of the wind"
     ) // Got these from the forums, kinda based -humboldt123 
-    // Got an updated version of them - Natan515
+     // Got an updated version of them - Natan515
```
Comment (Xiaro) [inline src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoEZ.kt:87]: Why is there an extra indent here now?

**Review by Xiaro — CHANGES_REQUESTED** [review]

File: src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoEZ.kt (hunk 2 above)
Comment (natan515) [reply src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoEZ.kt]: I'm not sure what you're talking about, I just added my own // as the last guy did.

Comment (humboldt123) [reply src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoEZ.kt]: There is an extra space though 
image

File: src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoEZ.kt [hunk 3]
```diff
@@ -63,13 +78,12 @@ object AutoEZ : Module() {
...
             "I love the way your hair glistens in the light",
-            "In my free time I like to watch cat videos on youtube",
             "When I saw the guy with a potion I knew there was trouble brewing.",
             "I enjoy long walks on the beach and playing Hypixel",
-            "Doin a bamboozle fren.",
             "I need help, teach me how to play!",
+            "What happens if I add chocolate milk to macaroni and cheese?",
             "Can you paint with all the colors of the wind"
-    ) // Got these from the forums, kinda based -humboldt123
+    ) // Got these from the forums, kinda based -humboldt123
```
Comment (humboldt123) [inline src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoEZ.kt:87]: You can add your thing back but don't have an extra *indent*

**Review by humboldt123 — CHANGES_REQUESTED** [review]

**Comment by humboldt123** [comment]
Fix requested changes, other than that, lgtm

File: src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoEZ.kt (hunk 2 above)
Comment (natan515) [reply src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoEZ.kt]: Ok ye but i just removed my line so it should be fine now right?

File: src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoEZ.kt (hunk 3 above)
Comment (natan515) [reply src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoEZ.kt:87]: Not sure what's the problem now after i just removed my line?

Comment (l1ving) [reply src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoEZ.kt:87]: no problem

**Review by l1ving — APPROVED** [review]

**Comment by l1ving** [comment]
thanks

**Pull request merged**